<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>
				MainWindow
			</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="1" colspan="3">
     <widget class="QDateEdit" name="endDateEdit">
      <property name="maximumDate">
       <date>
        <year>2100</year>
        <month>12</month>
        <day>31</day>
       </date>
      </property>
      <property name="minimumDate">
       <date>
        <year>2018</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="3">
     <widget class="QDateEdit" name="startDateEdit">
      <property name="maximumDate">
       <date>
        <year>2100</year>
        <month>12</month>
        <day>31</day>
       </date>
      </property>
      <property name="minimumDate">
       <date>
        <year>2018</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="endDateLabel">
      <property name="text">
       <string>End Date</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="userRemovePushButton">
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="4" colspan="3">
     <widget class="QListWidget" name="userListWidget"/>
    </item>
    <item row="0" column="0" rowspan="5">
     <widget class="QLabel" name="usersLabel">
      <property name="text">
       <string>Users</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="userAddPushButton">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="startDateLabel">
      <property name="text">
       <string>Start Date</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEditDragDrop" name="outputLineEdit"/>
    </item>
    <item row="7" column="1" colspan="3">
     <widget class="QGroupBox" name="metricTypeGroup">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="editingRadioButton">
         <property name="text">
          <string>Editing</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="qcRadioButton">
         <property name="text">
          <string>QC</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QLineEdit" name="userAddLineEdit"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="metricTypeLabel">
      <property name="text">
       <string>Metric Type</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="outputLabel">
      <property name="text">
       <string>Output Location</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2" colspan="2">
     <widget class="QPushButton" name="outputLocationSaveButton">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2" colspan="2">
     <widget class="QPushButton" name="runButton">
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QLineEditDragDrop</class>
   <extends>QLineEdit</extends>
   <header>chameleon.qt.qlineeditdragdrop</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
